incubator/hnc/e2e/tests/namespace_test.go

```diff
@@ -16,40 +16,33 @@ var _ = Describe("Namespace", func() {
 		prefix = namspacePrefix+"namespace-"
 	})
 
-	It("should create and delete a namespace", func() {
-		// Create a namespace "a"
-		cmd := exec.Command("kubectl", "create", "ns", prefix+"a")
-		Expect(cmd.Run()).Should(BeNil())
+	AfterEach(func() {
+		// clean up 
+		nses := []string {
+			prefix+"a",
+			prefix+"b",
+		}
+		cleanUpNses(nses)
```

Comment:
Maybe better to make `cleanupNamespaces` a variadic function?

`cleanupNamespaces(prefix+"a", prefix+"b")`

Reply:
While you're at it, instead of `prefix+"a"` and `prefix+"b"` everywhere, maybe just make these into constants like `nsA` and `nsB`?

Reply:
done